.github/workflows/debug_asan_clang.yml

```diff
@@ -66,6 +66,18 @@ jobs:
       # of the .github/actions/environment action.
       options: '--init --privileged'
 
+    strategy:
+      matrix:
+        ALLOCATOR: [sysmalloc, lj_alloc]
+        include:
+          - ALLOCATOR: sysmalloc
+            MAKE_CMD: test-debug-asan-sysmalloc
+          - ALLOCATOR: lj_alloc
```

Comment:
I would replace `lj_alloc` with `dlmalloc`. Or use `internal` instead `lj_alloc` and `external` instead `sysmalloc`.

Reply:
Good, replaced `lj_alloc` with `dlmalloc`.